### PR TITLE
[DSD-5618] Inji-certify svc Properties

### PR DESCRIPTION
**Pull request by VSIVAKALYAN**

### CODE REVIEW

File: certify-mock.properties [hunk 1]
```diff
@@ -0,0 +1,226 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
```
Comment (challabeehyv) [inline certify-mock.properties:4]: we don't need this file

Comment (challabeehyv) [inline certify-mock.properties:4]: we don't need this file

Comment (challabeehyv) [inline certify-mock.properties:4]: we don't need this file

**Review by challabeehyv — COMMENTED** [review]
can we also add https://github.com/mosip/inji-config/blob/qa/certify-mosipid-identity.properties

**Review by challabeehyv — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**